pkgs/development/python-modules/lsprotocol/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, attrs
+, buildPythonPackage
+, cattrs
+, fetchFromGitHub
+, flit-core
+, jsonschema
+, nox
+, pyhamcrest
+, pytest
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "lsprotocol";
+  version = "2022.0.0a9";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    hash = "sha256-6XecPKuBhwtkmZrGozzO+VEryI5wwy9hlvWE1oV6ajk=";
+  };
+
+  nativeBuildInputs = [
+    flit-core
+    nox
+  ];
+
+  propagatedBuildInputs = [
+    attrs
+    cattrs
+  ];
+
+  checkInputs = [
+    jsonschema
+    pyhamcrest
+    pytest
+  ];
```

The `lsprotocol` build is failing with `nox > Program pytest not found.` when running `nixpkgs-review`.

This seems to happen because of  #206742. It has to use `nativeCheckInputs` for `pytest` now. 

```suggestion
  nativeCheckInputs = [
    pytest
  ];

  checkInputs = [
    jsonschema
    pyhamcrest
  ];
```

---

Thanks, updated.